### PR TITLE
fix: updated user_function_deploy github workflow

### PR DESCRIPTION
**Pull request by silvicir**
#### Motivation and Context

Revert to the old deploy pipeline

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/user_function_deploy.yaml [hunk 1]
```diff
@@ -11,10 +11,12 @@ on:
 jobs:
   deploy:
     name: Deploy
-    uses: pagopa/dx/.github/workflows/web_app_deploy.yaml@add-web-app-deploy-workflow
+    uses: pagopa/dx/.github/workflows/function_app_deploy.yaml@main
```
Comment (gunzip) [inline .github/workflows/user_function_deploy.yaml:14]: why this change? cc @lucacavallaro

Comment (silvicir) [reply .github/workflows/user_function_deploy.yaml:14]: the new workflow is not working properly (maybe I'm missing something)

**Review by grausof — APPROVED** [review]
